src/views/InitialScreen/index.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+  <div class="tabview">
+    <div class="tab">
+      <button
+        class="tablinks"
+        :class="{ active: type == 'welcome' }"
+        @click="openTab($event, 'welcome')"
+      >
+        Welcome
+      </button>
+      <button
+        class="tablinks"
+        :class="{ active: type == 'intro' }"
+        @click="openTab($event, 'intro')"
+      >
+        Introduction
+      </button>
+      <button
+        class="tablinks"
+        :class="{ active: type == 'services' }"
+        @click="openTab($event, 'services')"
+      >
+        Services
+      </button>
+      <button
+        class="tablinks"
+        :class="{ active: type == 'docs' }"
+        @click="openTab($event, 'docs')"
+      >
+        Documentation
+      </button>
+    </div>
+    <div class="tabcontent">
+      <welcome v-if="type === 'welcome'" />
+      <introduction v-if="type === 'intro'" />
+      <docs v-if="type === 'docs'"></docs>
+      <services v-if="type === 'services'"></services>
+    </div>
+  </div>
+</template>
+<script>
+  import welcome from "./welcome.vue"
+  import introduction from "./intro.vue"
+  import docs from "./docs.vue"
+  import services from "./services.vue"
+  export default {
+    name: "InitialScreen",
+    components: {
+      welcome,
+      introduction,
+      docs,
+      services,
+    },
+    data() {
+      return {
+        tabcontent: "",
+        tablinks: "",
+        activeTab: false,
+        type: "welcome",
+        isActive: false,
+      }
+    },
+    methods: {
+      openTab(evt, tabName) {
+        this.activeTab = true
+        const tabs = ["welcome", "intro", "services", "docs"]
+        if (tabs.includes(tabName)) {
+          this.type = tabName
+          console.log("tab", tabName)
```

Comment:
remove console log added for testing purposes.

Reply:
Done